library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/CheckableWireframeMapper.kt

```diff
@@ -56,6 +54,15 @@ internal abstract class CheckableWireframeMapper<T>(
             null
         }
     }
+
+    internal open fun resolveCheckBoxShapeBorder(view: T, checkBoxColor: String):
```

Comment:
Maybe we can rename all the `resolveCheckBoxXxx` to `renameCheckableXxx`, otherwise it becomes weird that the RadioButtonMapper implements a `resolveCheckBoxShapeStyle` function